StudioCore/ParamEditor/ParamBank.cs

```diff
@@ -139,18 +139,21 @@ public static void LoadParamMeta(List<(string, PARAMDEF)> defPairs, AssetLocator
             }
         }
 
-        public CompoundAction LoadParamDefaultNames()
+        public CompoundAction LoadParamDefaultNames(string param = null, bool onlyAffectEmptyNames = false)
```

Comment:
Might make sense to use Directory.GetFiles() again with a different matcher, than a different function.